### ISSUE TITLE
add vim to base R docker image

### ISSUE THREAD

**Issue by emmjab** [SwissDataScienceCenter/renku-jupyter#56]
As an RStudio user who wants to use git via the terminal in jupyterlab (as opposed to the gui git interface provided by RStudio), it would be nice to have vim installed in the image.

**Comment by emmjab** [comment]
I tried rebuilding the r docker image that we have currently in the repo, and it failed with some conda issues. so ... we'll need to resolve that first before we can build a new image.

**Comment by rokroskar** [comment]
what were the conda issues?

**Comment by emmjab** [comment]
```
ChecksumMismatchError: Conda detected a mismatch between the expected content and downloaded content
for url 'https://repo.anaconda.com/pkgs/main/noarch/attrs-19.3.0-py_0.tar.bz2'.
  download saved to: /opt/conda/pkgs/attrs-19.3.0-py_0.tar.bz2
  expected sha256: 2463961334ca54ef0bfa0f592e2c2ca8555136ee14766e73caf7c49569d2898b
  actual sha256: 4ad3a1f3bd07e0f21f7e06e38e87d7a4538357722a5695fd24cdf4529a44d483
ChecksumMismatchError: Conda detected a mismatch between the expected content and downloaded content
for url 'https://repo.anaconda.com/pkgs/main/noarch/defusedxml-0.6.0-py_0.tar.bz2'.
  download saved to: /opt/conda/pkgs/defusedxml-0.6.0-py_0.tar.bz2
  expected sha256: f78d8a76e08f8a5676a8e11ee494c62c1e5d4e579c126a6481501f43dc6eca43
  actual sha256: 14a5bd7782acdc72edfd9db2170ca40ae2b0654333d8b463137d8b5b907d7cf7
```

**Comment by rokroskar** [comment]
that looks like something that should be temporary - did you retry?

**Comment by emmjab** [comment]
yeah maybe 10 times. i don't know if there's some kind of cache somewhere.

**Comment by rokroskar** [comment]
`/opt/conda/pkgs/`?

**Comment by emmjab** [comment]
Well, not sure what happened but everything's fine now!
